src/libtest/lib.rs

```diff
@@ -1484,7 +1484,10 @@ pub fn run_test(opts: &TestOpts,
 /// Fixed frame used to clean the backtrace with `RUST_BACKTRACE=1`.
 #[inline(never)]
 fn __rust_begin_short_backtrace<F: FnOnce()>(f: F) {
```

Comment:
This presumably no longer works now that the libstd implementation was updated?

Reply:
Yeah this is now broken. We could work around it by exporting `mark_start` as an unstable item.